Documentation/learning/glossary.md

```diff
@@ -26,10 +30,93 @@ Peer is another member of the same cluster.
 
 A proposal is a request (for example a write request, a configuration change request) that needs to go through raft protocol.
 
+## Quorum
+
+The number of members of a group/cluster required to be present to perform a set of instructions/operations, usually a majority.
```

Comment:
The number of members of a cluster required to be available to perform requests (except for local request when specified). Quorum is the majority (N/2 + 1) of the cluster. For example, when there are 5 members in the cluster, the quorum is 3. When there are 6 members, the quorum is still 3. When there are 7 members, the quorum is 4.